CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+# 4.1.0
+
+* Bump rubocop to version 0.76
+* Update the linter config to work with rubocop 0.76, updating UnneededCapitalW to UnneededCapitalW
```

Comment:
We probably don't need this, as it's a necessary change that people will get automatically, to make the upgrade seamless.